exercises/lens-person/description.md

```diff
@@ -2,8 +2,13 @@
 
 Use lenses to update nested records (specific to languages with immutable data).
 
-Updating fields of nested records is kind of annoying in Haskell. One solution
-is to use [lenses](https://wiki.haskell.org/Lens).  Implement several record
-accessing functions using lenses, you may use any library you want. The test
-suite also allows you to avoid lenses altogether so you can experiment with
-different approaches.
+Updating fields of nested, immutable records is kind of annoying. One could say that the
```

Comment:
I would also approve this without the words "One could say that". It didn't seem that the words significantly changed the meaning of this sentence or its impact on students.

Reply:
Good point. I don't mind removing that. Let me just understand your other point better first :)